CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,45 @@
+# ngrok Documentation Code of Conduct
```

Comment:
Remove `Documentation` here :)

Reply:
woops

Reply:
plus one to that

Reply:
fixed here: https://github.com/ngrok/ngrok-python/pull/45/commits/312762e6a6c57cefbc4dae24898531de6ecef8bd